doc/readme.md

```diff
@@ -325,7 +317,7 @@ cordova {plugin | plugins} [
 |       |--link                   | When installing from a local path, creates a symbolic link instead of copying files. The extent to which files are linked varies by platform. Useful for plugin development.
 |       |--nosave                 | Do NOT save the `<plugin-spec>` as part of the `plugin` element  into `config.xml` or `package.json`.
 |       |--force                  | _Introduced in version 6.1._ Forces copying source files from the plugin even if the same file already exists in the target directory.
-| remove `<pluginid>|<name>` [...]| | Remove plugins with the given IDs/name.
+| remove `<pluginid>` or remove `<name>` [...] | | Remove plugins with the given IDs/name.
```

Comment:
I'd prefer properly escaping the pipe

```suggestion
| remove `<pluginid>\|<name>` [...] | | Remove plugins with the given IDs/name.
```

Reply:
Definitely! I didn't know this was possible in Markdown.